client/my-sites/domains/domain-management/dataviews/use-bulk-action-notice.tsx

```diff
@@ -1,70 +1,123 @@
 import { JobStatus } from '@automattic/data-stores';
-import { StatusPopover } from '@automattic/domains-table/src/status-popover';
-import { TranslateOptionsPlural, useTranslate } from 'i18n-calypso';
+import { useIsEnglishLocale } from '@automattic/i18n-utils';
+import { hasTranslation } from '@wordpress/i18n';
+import { translate, useTranslate } from 'i18n-calypso';
 import { useCallback, useEffect, useMemo, useState } from 'react';
 import { useDispatch } from 'react-redux';
 import wpcomRequest from 'wpcom-proxy-request';
 import { successNotice, errorNotice } from 'calypso/state/notices/actions';
 import { useDomainsDataViewsContext } from './use-context';
 
-type ExistingReactNode = React.ReactElement | string | number;
-type TranslateFunction = (
-	original: string,
-	plural: string,
-	options: TranslateOptionsPlural
-) => ExistingReactNode;
+type TranslateFunction = typeof translate;
 
-const getSuccessMessage = ( job: JobStatus, translate: TranslateFunction ) => {
+const fallbackSuccessMessage = ( translate: TranslateFunction ) => {
+	return translate( 'Bulk domain updates finished successfully.' );
+};
+
+const getSuccessMessage = (
+	job: JobStatus,
+	translate: TranslateFunction,
+	isEnglishLocale: boolean
+) => {
 	if ( job.action !== 'set_auto_renew' ) {
-		return translate( 'Your domain has been updated.', 'Your domains have been updated.', {
-			count: job.success.length,
-		} );
+		if ( isEnglishLocale || hasTranslation( 'Your domain has been updated.' ) ) {
+			return translate( 'Your domain has been updated.', 'Your domains have been updated.', {
+				count: job.success.length,
+			} );
+		}
+
+		return fallbackSuccessMessage( translate );
 	}
 
 	// If the user tried to enable auto-renew:
 	if ( job.params.auto_renew ) {
+		if (
+			isEnglishLocale ||
+			hasTranslation( 'Automatic renewal has been enabled for your domain.' )
+		) {
+			return translate(
+				'Automatic renewal has been enabled for your domain.',
+				'Automatic renewal has been enabled for your domains.',
+				{ count: job.success.length }
+			);
+		}
+
+		return fallbackSuccessMessage( translate );
+	}
+
+	// If the user tried to disable auto-renew:
+	if (
+		isEnglishLocale ||
+		hasTranslation( 'Automatic renewal has been disabled for your domain.' )
```

Comment:
Don't we need a check for the plural version as well?

Reply:
The documentation of the function (in the doc-block) says we pass the singular version.